play/PlayerRules/CharacterCreation/02_Classes.txt

```diff
@@ -34,16 +34,18 @@ Allies must be able to hear and understand you to get the bonus.
 V
 Either 75% + (CHA mod) success or 30% added to next allied rolls.
 ----------------------
-It''ll heal [Min. Requirement of 7 fortitude]
-Make a FORT save at 50% to ignore an enemy damage. Remove one action next turn 
-to stare down your enemy, curse at them hitting you, make a badass remark, 
-etc. All damage taken during the encounter stacks, and half is taken at the 
-end of the encounter, regardless of your condition.
+Fight Feircely First
+The Soldier gets an additional +10 to initiative rolls. 
+In addition, the soldier with this class feat gets a +30% to crit against enemies
```

Comment:
I feel that it would be good to be explicit on whether the crit chance bonus operates by normal crit rules (crit doesn't apply to automatic fire weapons, etc.)